src/version.gradle

```diff
@@ -0,0 +1,57 @@
+// Copyright (c) 2020-2022, The Khronos Group Inc.
```

Comment:
Figured out the problem here - the scripts that I use to turn the single internal repo into the various public repos, wasn't including this file in the CTS tarball when it should have. I've corrected that now internally so this won't happen again. If this is the same as the version.gradle in the other repos, I'll just merge this one from you and keep it simple :)